app/src/androidTest/java/org/mozilla/fenix/ui/EnhancedTrackingProtectionTest.kt

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ui
+
+import androidx.core.net.toUri
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.fenix.helpers.HomeActivityTestRule
+import org.mozilla.fenix.ui.robots.enhancedTrackingProtection
+import org.mozilla.fenix.ui.robots.homeScreen
+import org.mozilla.fenix.ui.robots.navigationToolbar
+import org.mozilla.fenix.ui.robots.settingsSubMenuEnhancedTrackingProtection
+
+/**
+ *  Tests for verifying basic UI functionality of Enhanced Tracking Protection
+ *
+ *  Including
+ *  - Verifying default states
+ *  - Verifying Enhanced Tracking Protection notification bubble
+ *  - Verifying Enhanced Tracking Protection notification shield
+ *  - Verifying Enhanced Tracking Protection content sheet
+ *  - Verifying Enhanced Tracking Protection content sheet details
+ *  - Verifying Enhanced Tracking Protection toggle
+ *  - Verifying Enhanced Tracking Protection site exceptions
+ */
+
+class EnhancedTrackingProtectionTest {
+    // Uses a variety of hosted tracking test URLs
+    private object EnhancedTrackingProtectionURLs {
+        const val ads = "https://mozilla.github.io/tracking-test/ads.html"
```

Comment:
I'm not  sure, if it could be a better idea to have these resources locally, as this could introduce some flakiness to the test suit, as this resource could change at any time or be unavailable.

Reply:
I've been investigating this but am currently blocked on our MockResponse and it's handling of images (also don't know how ETP works off localhost).

Reply:
Updated to local page using (https://github.com/englehardt/englehardt.github.io/tree/master/test/trackingprotection/test_pages) which use dummytracker (https://github.com/mozilla/dummytracker) resources for list checks.